charts/argus/values.schema.json

```diff
@@ -2349,6 +2349,49 @@
       "$id": "#/properties/securityContext",
       "$ref": "#/definitions/io.k8s.api.core.v1.SecurityContext"
     },
+    "helmChartReleases": {
+      "$comment": "tf:optional",
+      "$id": "#/properties/helmChartReleases",
+      "type": "object",
+      "title": "Configurations related to helm charts releases monitoring",
+      "description": "The Helm Charts releases Monitoring settings",
+      "default": {},
+      "examples": [
+        ""
+      ],
+      "required": [
+      ],
+      "properties": {
+        "ops_notes": {
```

Comment:
Plz don't deviate from `camelCase` structure.

Reply:
refactored it